yaml/_dumper.ts

```diff
@@ -622,8 +622,7 @@ function writeFlowSequence(
 function writeBlockSequence(
   state: DumperState,
   level: number,
-  // deno-lint-ignore no-explicit-any
-  object: any,
+  object: unknown[],
```

Comment:
Again, this argument could be removed and we could instead use `this.dump` within the method.